zrml/neo-swaps/src/tests/buy.rs

```diff
@@ -243,14 +243,15 @@ fn buy_fails_on_asset_not_found(market_type: MarketType) {
 }
 
 #[test]
-fn buy_fails_on_numerical_limits() {
+fn buy_fails_if_amount_in_is_greater_than_numerical_threshold() {
```

Comment:
Can we assert for the numerical threshold here explicitly, so that the reader knows, what the numerical threshold is?

Reply:
012e595e0efa32b4be0865bf4a4a9831bc36c761